### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by mvorisek**

### CODE REVIEW

File: .github/workflows/unit-tests.yml [hunk 1]
```diff
@@ -129,7 +129,7 @@ jobs:
 
       - name: Install PHP dependencies
         run: |
-          if [[ "${{ matrix.type }}" != Phpunit* ]; then composer remove --no-interaction --no-update phpunit/phpunit johnkary/phpunit-speedtrap phpunit/dbunit phpunit/phpcov --dev ; fi
+          if [[ "${{ matrix.type }}" != Phpunit* ]]; then composer remove --no-interaction --no-update phpunit/phpunit johnkary/phpunit-speedtrap phpunit/dbunit phpunit/phpcov --dev ; fi
```
Comment (georgehristov) [inline .github/workflows/unit-tests.yml]: Is the typo not the `[[`?

Comment (mvorisek) [reply .github/workflows/unit-tests.yml]: no, because of `*`, bacause of bash...

**Review by georgehristov — APPROVED** [review]
Any idea why tests are failing?

File: .github/workflows/unit-tests.yml (hunk 1 above)
Comment (mvorisek) [reply .github/workflows/unit-tests.yml]: failing because GH probably uses sh instead of bash, debugging it....

**Pull request merged**